src/test/java/com/kodilla/ecommercee/domain/CartTestSuite.java

```diff
@@ -0,0 +1,190 @@
+package com.kodilla.ecommercee.domain;
+
+import com.kodilla.ecommercee.repository.CartRepository;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import javax.transaction.Transactional;
+import java.math.BigDecimal;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@Transactional
+public class CartTestSuite {
+
+    @Autowired
+    private CartRepository cartRepository;
+
+    @Test
+    public void shouldCreateEmptyCart() {
+        //Given
+        Cart cart = new Cart(new BigDecimal("0"));
+        List<Product> products = cart.getProducts();
+
+        //When
+        cartRepository.save(cart);
+        long cartId = cart.getCartId();
+
+        //Then
+        BigDecimal value = new BigDecimal("0");
+
+        assertNotEquals(0, cartId);
+        assertEquals(0, products.size());
+        assertEquals(value, cart.getValue());
+    }
+
+    @Test
+    public void shouldAddOneProduct() {
+        //Given
+        Group group = new Group("Toys");
+        Product product = new Product("Teddy Bear", "little plush toy", new BigDecimal("10"));
+        group.getProducts().add(product);
+        product.getGroupsOfProduct().add(group);
+
+        Cart cart = new Cart(new BigDecimal("0"));
+        cart.getProducts().add(product);
+        cart.setValue(new BigDecimal("10"));
+        List<Product> products = cart.getProducts();
+
+        //When
+        cartRepository.save(cart);
+        long cartId = cart.getCartId();
+
+        //Then
+        assertNotEquals(0, cartId);
+        assertEquals(1, products.size());
```

Comment:
`products` to jest lista stworzna wcześniej przez Ciebie. Ona na pewno ma jeden element. Powinnaś sprawdzać czy lista pobrana z bazy (z bazy pobieramy `cart`, a z tego `cart` listę produktów) ma odpowiednie elementy